docs/source/frameworks/picklable.rst

```diff
@@ -2,41 +2,46 @@
 Picklable Model
 ===============
 
-Here's an example of saving any Python object or function as model, and create a runner
-instance:
+For custom ML models created with pure Python code, a simple way to make it work with
+the BentoML workflow is via ``bentoml.picklable_model``.
+
+Here's an example of saving a simple Python function as model:
 
 .. code:: python
 
+    from typing import List
+    import numpy as np
     import bentoml
 
-    class MyPicklableModel:
-        def predict(self, some_integer: int):
-            return some_integer ** 2
+    def my_python_model(input_list: List[int]) -> List[int]:
+        return np.square(np.array(input_list))
 
-    # `save` a given model or function
-    model = MyPicklableModel()
-    tag = bentoml.picklable_model.save_model(
-        'mypicklablemodel',
-        model,
-        signatures={"predict": {"batchable": False}}
+    # `save_model` saves a given python object or function
+    saved_model = bentoml.picklable_model.save_model(
+        'my_python_model',
+        my_python_model,
+        signatures={"__call__": {"batchable": True}}
     )
+    print(f"Model saved: {saved_model}")
 
-    # retrieve metadata with `bentoml.models.get`:
-    metadata = bentoml.picklable_model.get(tag)
 
-    # load the model back:
-    loaded = bentoml.picklable_model.load_model("mypicklablemodel:latest")
+Load the model back to memory for testing:
 
-    # Run a given model under `Runner` abstraction with `to_runner`
-    runner = bentoml.picklable_model.get(tag).to_runner()
-    runner.init_local()
-    runner.predict.run(7)
+.. code:: python
+
+    In [1]: loaded_model = bentoml.picklable_model.load_model("my_python_model:latest")
 
+    In [2]: loaded_model([1, 2, 3])
+    Out[2]: array([1, 4, 9])
```

Comment:
```suggestion
    loaded_model = bentoml.picklable_model.load_model("my_python_model:latest")

    loaded_model([1, 2, 3])
    # out: array([1, 4, 9])
```
We should be consistent with all the code-block. If we want to make this as a demonstration block the we might as well use doctest formatted code block.

Reply:
food for thought.